components/blitz/resources/omero/api/IAdmin.ice

```diff
@@ -18,52 +18,428 @@ module omero {
+                /**
+                 * Fetches all {@link omero.model.Experimenter} users
+                 * contained in this group. The returned users will have all
+                 * fields filled in and all collections unloaded.
+                 *
+                 * @param groupId id of the ExperimenterGroup
+                 * @return non-null array of all
+                 * {@link omero.model.Experimenter} users in this group.
+                 */
                 idempotent ExperimenterList containedExperimenters(long groupId) throws ServerError;
+
+                /**
+                 * Fetches all {@link omero.model.ExperimenterGroup} groups of
+                 * which the given user is a member. The returned groups will
+                 * have all fields filled in and all collections unloaded.
+                 *
+                 * @param experimenterId id of the Experimenter. Not null.
+                 * @return non-null array of all
+                 * {@link omero.model.ExperimenterGroup} groups for this user.
+                 */
                 idempotent ExperimenterGroupList containedGroups(long experimenterId) throws ServerError;
+
+                /**
+                 * Retrieves the default {@link omero.model.ExperimenterGroup}
+                 * group for the given user id.
+                 *
+                 * @param experimenterId of the Experimenter. Not null.
+                 * @return non-null {@link omero.model.ExperimenterGroup}. If
+                 *         no default group is found, an exception will be
+                 *         thrown.
+                 */
                 idempotent omero::model::ExperimenterGroup getDefaultGroup(long experimenterId) throws ServerError;
+
+                /**
+                 * Looks up {@link omero.model.Experimenter} experimenters who
+                 * use LDAP authentication  (has set dn on password table).
+                 *
+                 * @return Experimenter. Never null.
+                 */
                 idempotent string lookupLdapAuthExperimenter(long id) throws ServerError;
+
+                /**
+                 * Looks up all ids of {@link omero.model.Experimenter}
+                 * experimenters who use LDAP authentication (has set dn on
+                 * password table).
+                 *
+                 * @return list of experimenters. Never null.
+                 */
                 idempotent RList lookupLdapAuthExperimenters() throws ServerError;
+
+                /**
+                 * Finds the ids for all groups for which the given
+                 * {@link omero.model.Experimenter} is a member.
+                 *
+                 * @param e Non-null, managed (i.e. with id)
+                 * @see omero.model.Details#getOwner
+                 */
                 idempotent LongList getMemberOfGroupIds(omero::model::Experimenter exp) throws ServerError;
```

Comment:
There are also quite a few places where no return is documented. Again, is this something to be fixed now?